pkg/controller/endpoint/endpoints_controller.go

```diff
@@ -534,8 +534,8 @@ func (e *Controller) syncService(key string) error {
 		delete(newEndpoints.Annotations, v1.EndpointsLastChangeTriggerTime)
 	}
 
-	if overCapacity(newEndpoints.Subsets) {
-		newEndpoints.Annotations[v1.EndpointsOverCapacity] = "warning"
+	if truncateEndpoints(newEndpoints) {
+		newEndpoints.Annotations[v1.EndpointsOverCapacity] = "truncated"
```

Comment:
Tiny Nit: "truncated" feels like something that should be a const somewhere in the controller. (Not sure why I didn't do this to start with).

Reply:
I was 50/50 since it was only used in this one place. Fixed in the newest patch.